int32 targetId
float64 x
float64 y
float64 vx
float64 vy
float64[16] covariances